task3/js/app.js

```diff
@@ -0,0 +1,117 @@
+document.addEventListener('DOMContentLoaded', () => {
+
+    let index = 0;
+    const zombieAll = [];
+
+    function zombieCreate(zombie, img) {
```

Comment:
nope
this task is about classes and inheritance

you have a lot of code in one file and this is not good

define a base function-constructor for zombie
define a function-constructor for each zombie type

base zombie class should implement all logic with adding the DOM elements